pkg/kubectl/cmd/util/factory.go

```diff
@@ -240,7 +240,7 @@ type ObjectMappingFactory interface {
 	OpenAPISchema() (openapi.Resources, error)
 }
 
-// BuilderFactory holds the second level of factory methods.  These functions depend upon ObjectMappingFactory and ClientAccessFactory methods.
+// BuilderFactory holds the third level of factory methods. These functions depend upon ObjectMappingFactory and ClientAccessFactory methods.
```

Comment:
Why is `third` here? I must be missing something :)

Reply:
ClientAccessFactory is the first level.
ObjectMappingFactory depends on ClientAccessFactory and it's the second level.
BuilderFactory depends on ObjectMappingFactory, so I think it should be the third level.

Reply:
yes, rings are `ClientAccessFactory -> ObjectMappingFactory -> BuilderFactory`, thanks for the comment fix